source/features/add-releases-tab.js

```diff
@@ -30,6 +30,8 @@ function updateReleasesCount() {
 }
 
 export default async () => {
+	registerShortcut('repos', 'g r', 'Go to Releases');
```

Comment:
I don't think this should be added so early. If the features don't exist/work then maybe they shouldn't appear in there.